### PR TITLE
See also: remove text, add modules

### PR DESCRIPTION
**Pull request by estelle**
per : https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section, removed "mapped logical properties" text in see also sections. 
ordered the see also in some cases, also added the module or other related props when missing.

### CODE REVIEW

File: files/en-us/web/css/overscroll-behavior-x/index.md [hunk 1]
```diff
@@ -76,5 +76,8 @@ main > div {
 
 ## See also
 
-- [Take control of your scroll: customizing pull-to-refresh and overflow effects](https://developer.chrome.com/blog/overscroll-behavior/#full-demo)
-- The mapped logical properties: {{cssxref("overscroll-behavior-inline")}}, {{cssxref("overscroll-behavior-block")}}
+- {{cssxref("overscroll-behavior-x")}}
```
Comment (dipikabh) [inline files/en-us/web/css/overscroll-behavior-x/index.md:79]: this is self-referencing
```suggestion
- {{cssxref("overscroll-behavior")}}
```

File: files/en-us/web/css/padding-bottom/index.md [hunk 2]
```diff
@@ -78,5 +78,7 @@ The `padding-bottom` property is specified as a single value chosen from the lis
 ## See also
 
 - [Introduction to the CSS basic box model](/en-US/docs/Web/CSS/CSS_box_model/Introduction_to_the_CSS_box_model)
```
Comment (dipikabh) [inline files/en-us/web/css/padding-bottom/index.md:80]: can be moved to the bottom of the list

File: files/en-us/web/css/padding-left/index.md [hunk 3]
```diff
@@ -76,5 +76,7 @@ The `padding-left` property is specified as a single value chosen from the list
 ## See also
 
 - [Introduction to the CSS basic box model](/en-US/docs/Web/CSS/CSS_box_model/Introduction_to_the_CSS_box_model)
```
Comment (dipikabh) [inline files/en-us/web/css/padding-left/index.md:78]: can be moved to the bottom of the list

File: files/en-us/web/css/padding-right/index.md [hunk 4]
```diff
@@ -76,5 +76,7 @@ The `padding-right` property is specified as a single value chosen from the list
 ## See also
 
 - [Introduction to the CSS basic box model](/en-US/docs/Web/CSS/CSS_box_model/Introduction_to_the_CSS_box_model)
```
Comment (dipikabh) [inline files/en-us/web/css/padding-right/index.md:78]: can be moved to the bottom of the list

File: files/en-us/web/css/padding-top/index.md [hunk 5]
```diff
@@ -78,5 +78,7 @@ The `padding-top` property is specified as a single value chosen from the list b
 ## See also
 
 - [Introduction to the CSS basic box model](/en-US/docs/Web/CSS/CSS_box_model/Introduction_to_the_CSS_box_model)
```
Comment (dipikabh) [inline files/en-us/web/css/padding-top/index.md:80]: can be moved to the bottom of the list

File: files/en-us/web/css/padding-bottom/index.md (hunk 2 above)
Comment (dipikabh) [reply files/en-us/web/css/padding-bottom/index.md:80]: The box model module link can be added to all the padding pages

File: files/en-us/web/css/padding/index.md [hunk 6]
```diff
@@ -131,4 +131,5 @@ padding: 1em 3px 30px 5px; /* top:    1em padding  */
 
 - [Introduction to the CSS basic box model](/en-US/docs/Web/CSS/CSS_box_model/Introduction_to_the_CSS_box_model)
```
Comment (dipikabh) [inline files/en-us/web/css/padding/index.md:132]: can be moved to the bottom of the list

File: files/en-us/web/css/padding/index.md [hunk 7]
```diff
@@ -131,4 +131,5 @@ padding: 1em 3px 30px 5px; /* top:    1em padding  */
...
 - {{cssxref("padding-top")}}, {{cssxref("padding-right")}}, {{cssxref("padding-bottom")}}, and {{cssxref("padding-left")}}.
```
Comment (dipikabh) [inline files/en-us/web/css/padding/index.md:133]:
```suggestion
- {{cssxref("padding-top")}}, {{cssxref("padding-right")}}, {{cssxref("padding-bottom")}}, and {{cssxref("padding-left")}}
```

File: files/en-us/web/css/right/index.md [hunk 8]
```diff
@@ -167,6 +167,8 @@ div {
 
 ## See also
 
-- {{cssxref("inset")}}, the shorthand for all related properties: {{cssxref("top")}}, {{cssxref("bottom")}}, and {{cssxref("left")}}
-- The mapped logical properties: {{cssxref("inset-block-start")}}, {{cssxref("inset-block-end")}}, {{cssxref("inset-inline-start")}}, and {{cssxref("inset-inline-end")}} and the shorthands {{cssxref("inset-block")}} and {{cssxref("inset-inline")}}
+- {{cssxref("top")}}, {{cssxref("bottom")}}, and {{cssxref("left")}}
+- {{cssxref("inset")}} shorthand
+- {{cssxref("inset-block-start")}}, {{cssxref("inset-block-end")}}, {{cssxref("inset-inline-start")}}, and {{cssxref("inset-inline-end")}}
+- {{cssxref("inset-block")}} and {{cssxref("inset-inline")}} shorthands
 - {{cssxref("position")}}
```
Comment (dipikabh) [inline files/en-us/web/css/right/index.md:174]: Link to the https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout module can be added on `right` and `top` pages

File: files/en-us/web/css/width/index.md [hunk 9]
```diff
@@ -186,5 +186,5 @@ p.minblue {
 - {{cssxref("height")}}
 - {{cssxref("box-sizing")}}
 - {{cssxref("min-width")}}, {{cssxref("max-width")}}
-- The mapped logical properties: {{cssxref("block-size")}}, {{cssxref("inline-size")}}
+- {{cssxref("block-size")}}, {{cssxref("inline-size")}}
 - {{cssxref("anchor-size()")}}
```
Comment (dipikabh) [inline files/en-us/web/css/width/index.md:190]: [The box model](/en-US/docs/Web/CSS/CSS_box_model/Introduction_to_the_CSS_box_model) can be moved to the end with an update to the link text.
And another link to box model module can be added

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @estelle for updating the see also's on these pages.

I've highlighted some links that can be added and some that can be reordered.

**Comment by estelle** [comment]
@dipikabh Thanks for the review

File: files/en-us/web/css/inset/index.md [hunk 10]
```diff
@@ -105,6 +105,8 @@ div {
 
 ## See also
 
-- The longhand box offset properties: {{cssxref("top")}}, {{cssxref("right")}}, {{cssxref("bottom")}}, and {{cssxref("left")}}.
-- The mapped logical shorthands: {{cssxref("inset-block")}} and {{cssxref("inset-inline")}}
-- The {{cssxref("margin")}} shorthand multi-value syntax.
+- {{cssxref("top")}}, {{cssxref("right")}}, {{cssxref("bottom")}}, and {{cssxref("left")}}.
```
Comment (estelle) [inline files/en-us/web/css/inset/index.md:108]:
```suggestion
- {{cssxref("top")}}, {{cssxref("right")}}, {{cssxref("bottom")}}, and {{cssxref("left")}}
```

File: files/en-us/web/css/margin-left/index.md [hunk 11]
```diff
@@ -190,5 +190,7 @@ Percentage values for `margin-left` are relative to the container's inline size.
 
 ## See also
 
-- {{cssxref("margin-top")}}, {{cssxref("margin-right")}}, and {{cssxref("margin-bottom")}} and the {{cssxref("margin")}} shorthand
-- The mapped logical properties: {{cssxref("margin-block-start")}}, {{cssxref("margin-block-end")}}, {{cssxref("margin-inline-start")}}, and {{cssxref("margin-inline-end")}} and the shorthands {{cssxref("margin-block")}} and {{cssxref("margin-inline")}}
+- {{cssxref("margin-top")}}, {{cssxref("margin-right")}}, and {{cssxref("margin-bottom")}}
+- {{cssxref("margin")}} shorthand
+- {{cssxref("margin-block-start")}}, {{cssxref("margin-block-end")}}, {{cssxref("margin-inline-start")}}, and {{cssxref("margin-inline-end")}}
+- {{cssxref("margin-block")}} and {{cssxref("margin-inline")}} shorthands
```
Comment (dipikabh) [inline files/en-us/web/css/margin-left/index.md:196]:
```suggestion
- {{cssxref("margin-block")}} and {{cssxref("margin-inline")}} shorthands
- [CSS box model](/en-US/docs/Web/CSS/CSS_box_model) module
```

File: files/en-us/web/css/margin-right/index.md [hunk 12]
```diff
@@ -169,5 +169,7 @@ The `margin-right` property is specified as the keyword `auto`, or a `<length>`,
 
 ## See also
 
-- {{cssxref("margin-top")}}, {{cssxref("margin-bottom")}}, and {{cssxref("margin-left")}} and the {{cssxref("margin")}} shorthand
-- The mapped logical properties: {{cssxref("margin-block-start")}}, {{cssxref("margin-block-end")}}, {{cssxref("margin-inline-start")}}, and {{cssxref("margin-inline-end")}} and the shorthands {{cssxref("margin-block")}} and {{cssxref("margin-inline")}}
+- {{cssxref("margin-top")}}, {{cssxref("margin-bottom")}}, and {{cssxref("margin-left")}}
+- {{cssxref("margin")}} shorthand
+- {{cssxref("margin-block-start")}}, {{cssxref("margin-block-end")}}, {{cssxref("margin-inline-start")}}, and {{cssxref("margin-inline-end")}}
+- {{cssxref("margin-block")}} and {{cssxref("margin-inline")}} shorthands
```
Comment (dipikabh) [inline files/en-us/web/css/margin-right/index.md:175]:
```suggestion
- {{cssxref("margin-block")}} and {{cssxref("margin-inline")}} shorthands
- [CSS box model](/en-US/docs/Web/CSS/CSS_box_model) module
```

File: files/en-us/web/css/margin-top/index.md [hunk 13]
```diff
@@ -80,5 +80,7 @@ The `margin-top` property is specified as the keyword `auto`, or a `<length>`, o
 
 ## See also
 
-- {{cssxref("margin-right")}}, {{cssxref("margin-bottom")}}, and {{cssxref("margin-left")}} and the {{cssxref("margin")}} shorthand
-- The mapped logical properties: {{cssxref("margin-block-start")}}, {{cssxref("margin-block-end")}}, {{cssxref("margin-inline-start")}}, and {{cssxref("margin-inline-end")}} and the shorthands {{cssxref("margin-block")}} and {{cssxref("margin-inline")}}
+- {{cssxref("margin-right")}}, {{cssxref("margin-bottom")}}, and {{cssxref("margin-left")}}
+- {{cssxref("margin")}} shorthand
+- {{cssxref("margin-block-start")}}, {{cssxref("margin-block-end")}}, {{cssxref("margin-inline-start")}}, and {{cssxref("margin-inline-end")}}
+- {{cssxref("margin-block")}} and {{cssxref("margin-inline")}} shorthands
```
Comment (dipikabh) [inline files/en-us/web/css/margin-top/index.md:86]:
```suggestion
- {{cssxref("margin-block")}} and {{cssxref("margin-inline")}} shorthands
- [CSS box model](/en-US/docs/Web/CSS/CSS_box_model) module
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Review by dipikabh — APPROVED** [review]
Thanks, @estelle!

**Pull request merged**